ecdsa/Cargo.toml

```diff
@@ -34,6 +34,7 @@ default = ["digest"]
 alloc = ["elliptic-curve/alloc", "signature/alloc"]
 std = ["alloc", "elliptic-curve/std", "signature/std"]
 
+algorithm-identifier = ["elliptic-curve/pkcs8"]
```

Comment:
Since it needs `pkcs8` anyway, why not just gate this on `pkcs8`?

Reply:
I thought that is not that logical, as one is not directly related to another, but why not?
I used `pkcs8` as a way to get `spki` without adding direct dependency.